src/exporter/trace/mod.rs

```diff
@@ -85,6 +85,9 @@ pub struct SpanData {
     pub status_message: String,
     /// Resource contains attributes representing an entity that produced this span.
     pub resource: Arc<sdk::Resource>,
+    /// Instrumentation library that produced this span
+    #[cfg_attr(feature = "serialize", serde(skip))]
```

Comment:
Why skip this?

Reply:
InstrumentationLibrary uses borrowed strings which causes issues for serde Deserialize.

Additionally avoids any unintentional side effects of this change

Reply:
ok, can revisit if this causes issues for anyone who wants to depend on the serialized version at some point. Seems fine for now as the binary formats are still in flux.